src/main/java/org/softc/armoryexpansion/common/integration/aelib/plugins/tinkersconstruct/material/ToolMaterial.java

```diff
@@ -0,0 +1,85 @@
+package org.softc.armoryexpansion.common.integration.aelib.plugins.tinkersconstruct.material;
+
+import org.softc.armoryexpansion.common.integration.aelib.config.MaterialConfigOptions;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.general.material.BasicMaterial;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.library.materials.ExtraMaterialStats;
+import slimeknights.tconstruct.library.materials.HandleMaterialStats;
+import slimeknights.tconstruct.library.materials.HeadMaterialStats;
+import slimeknights.tconstruct.library.materials.MaterialTypes;
+
+public class ToolMaterial extends BasicMaterial implements IToolMaterial {
+    protected HeadMaterialStats headMaterialStats;
+    protected HandleMaterialStats handleMaterialStats;
+    protected ExtraMaterialStats extraMaterialStats;
+
+    @Override
+    public HeadMaterialStats getHeadMaterialStats() {
+        return this.headMaterialStats;
+    }
+
+    @Override
+    public HandleMaterialStats getHandleMaterialStats() {
+        return this.handleMaterialStats;
+    }
+
+    @Override
+    public ExtraMaterialStats getExtraMaterialStats() {
+        return this.extraMaterialStats;
+    }
+
+    @Override
+    public boolean isToolMaterial() {
+        return null != this.headMaterialStats
+                || null != this.handleMaterialStats
+                || null != this.extraMaterialStats;
+    }
+
+    @Override
+    public boolean isArmorMaterial() {
+        return false;
+    }
+
+    @Override
+    public boolean isRangedMaterial() {
+        return false;
+    }
+
+    @Override
+    public boolean registerTinkersMaterialStats(MaterialConfigOptions properties){
+        if (properties.materialEnabled() && properties.isToolEnabled()) {
+            slimeknights.tconstruct.library.materials.Material material = TinkerRegistry.getMaterial(this.getIdentifier());
+//            if (!"unknown".equals(material.getIdentifier())) {
+                this.registerToolStats(material, properties);
+                return true;
+//            }
+        }
+        return false;
+    }
+
+    void registerToolStats(slimeknights.tconstruct.library.materials.Material material, MaterialConfigOptions properties){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)